Source/gmenu.cpp

```diff
@@ -291,9 +291,6 @@ int __fastcall gmenu_get_lfont(TMenuItem *pItem)
 
 int __fastcall gmenu_presskeys(int a1)

```

Comment:
I've modified function so that it returns BOOLs and changed the return values accordingly, it is still bin exact.

Reply:
> The left/right and up/down functions where probably also BOOLs, however IMHO it would probably make the code easier to read if we came up with specific typedefs for left/right and up/down so that you see that immediately when looking at the call.

Sounds like a good idea. If you look at this, you could also change the parameter name `a1` to `vkey`, to be consistent with the call site in diablo.cpp.

From https://github.com/diasurgical/devilution/blob/62fca0f486e8bea79db16fa7a030fa96663f4df4/Source/diablo.cpp#L1055:

```c
	if (!gmenu_presskeys(vkey) && !control_presskeys(v1)) {
```